src/TransformationValidator.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\TransformationMigrate;
+
+use Keboola\TransformationMigrate\Exception\CheckConfigException;
+
+class TransformationValidator
+{
+    private array $config;
+
+    public function __construct(array $config)
+    {
+        $this->config = $config;
+    }
+
+    public function validate(): void
+    {
+        $this->validateBackend();
+        $this->validatePhases();
+    }
+
+    private function validateBackend(): void
+    {
+        $backends = array_map(
+            fn(array $v) => sprintf('%s-%s', $v['configuration']['backend'], $v['configuration']['type']),
+            $this->config['rows']
+        );
+        $uniqueBackends = array_unique($backends);
+
+        if (count($uniqueBackends) > 1) {
```

Comment:
Moze sa stat, ze  `$uniqueBackends = 0` ale ma transformacia vzdy aspon jeden row?
Ide mi o to ci vies zmigrovat aj prazdnu transformaciu, ... tie stare nepoznam uplne podrobne, tak neviem ci to moze nastat.

Reply:
`$uniqueBackends` je vždycky minimálně 1 pokud má alespoň jeden row

Reply:
ok, a moze nastat situacia, ze nema ziaden row?

Reply:
to samozřejmě může.... to bychom mohli taky checknout 👍